SECP-service/src/main/java/com/visucius/secp/resources/GroupResource.java

```diff
@@ -0,0 +1,44 @@
+package com.visucius.secp.resources;
+
+
+import com.codahale.metrics.annotation.Timed;
+import com.visucius.secp.Controllers.GroupController;
+import com.visucius.secp.DTO.GroupRequest;
+import io.dropwizard.auth.Auth;
+import io.dropwizard.hibernate.UnitOfWork;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+@Path("/groups")
+@RolesAllowed("ADMIN")
+public class GroupResource {
+
+    private GroupController groupController;
+
+    public GroupResource(GroupController controller)
+    {
+        this.groupController = controller;
+    }
+
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Timed
+    @UnitOfWork
+    public Response create(@Auth GroupRequest request) {
+        return groupController.createGroup(request);
+    }
+
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Timed
+    @UnitOfWork
+    @Path("/modify/{id}")
+    public Response modify(@Auth GroupRequest request, @PathParam("id") int id)
```

Comment:
It would be better if this was split into /modify/id/roles and /modify/id/permissionLevels. It's your call if you want to change it or not :)